specs-go/config.go

```diff
@@ -79,6 +85,8 @@ type Platform struct {
 	OS string `json:"os"`
 	// Arch is the architecture
 	Arch string `json:"arch"`
+	// OSVersion is the version of the operating system. (this field is platform dependent)
+	OSVersion string `json:"os.version,omitempty" platform:"windows"`
```

Comment:
On Wed, Jun 22, 2016 at 09:25:09AM -0700, John Howard wrote:
> > +	// OSVersion is the version of the operating system. (this field is platform dependent)
> > +	OSVersion string `json:"os.version,omitempty" platform:"windows"`
> 
> We use it today in docker/docker for difference in functionality
> between versions of Windows as opposed to whether things are
> supported or not supported.

Grepping turned up [1,2].  I still think that is a runtime-flag
concern (especially if process.terminal is a runtime-flag concern
[3]).  Can't the code that currently sets OSVersion just switch over
to setting a runtime flag?  It doesn't seem like that would be a big
change.

[1]: https://github.com/docker/docker/blob/v1.12.0-rc2/libcontainerd/container_windows.go#L125-L126
[2]: https://github.com/docker/docker/blob/v1.12.0-rc2/libcontainerd/process_windows.go#L34-L51
[3]: https://github.com/opencontainers/runtime-spec/issues/494#issuecomment-225251219